wrapper/wrapper.go

```diff
@@ -258,7 +271,10 @@ func WrapRequest(obj interface{}) http.HandlerFunc {
 			if err != nil {
 				data = []byte(`{}`)
 			}
-			http.Error(w, string(data), http.StatusBadRequest)
+			w.Header().Set("content-type", "application/json")
+			w.Header().Set("X-Content-Type-Options", "nosniff")
```

Comment:
What does it do ?

Reply:
![image](https://user-images.githubusercontent.com/114915010/199527353-ecd05087-f4aa-48bc-af40-50c4c4b00cbf.png)
http.Error(...) return plain text and I want JSON since we are Marshalling it, just took  `X-Content-Type-Options` from source code.